src/libraries/formatUtils/optionFormatting.ts

```diff
@@ -0,0 +1,18 @@
+import { MedicalTypeDTO } from "../../generated";
+
+export const medicalTypesFormatter = (
+  medicalTypes: MedicalTypeDTO[] | undefined
+): { value: string; label: string }[] => {
+  let options: { value: string; label: string }[] = [];
+  if (medicalTypes) {
+    medicalTypes.forEach((el) => {
+      let option: { value: string; label: string } = {
```

Comment:
Hi, 
first of all I am sorry we accidentally created a pull request, was not our intention since everything is still in progress.
Anyway your feedbacks are being really useful to improve the code earlier.
In this case, as for the other comment
"medicalTypesOptions and medicalTypes are duplication. I would suggest to use only one if needed."

The problem is we need to convert the MedicalTypeDTO
_{ code: string, description: string }_
in a 'Option' for the SelectField  
_{ value: string, label: string }_

Do you have any advise?